Changed saving of colors as strings to an rgb tuple (#73)
sunpy/sunkit-pyvista: changelog/73.trivial.rst

> Changes the way colors are saved as well as default colors of meshes.

Reviewer: ```suggestion
Changed the manner that colors or colormaps are saved.
Changed default of meshes to be white.
```
